### PR TITLE
Improve GitHub actions

### PR DESCRIPTION
**Pull request by UBaggeler**
- Add Swiftlint as separate action
- Publish test results in pull-request as comment
- Add a badge to README.me

### CODE REVIEW

File: Sources/CrowdNotifierBaseSDK/Models/ProblematicEventInfo.swift [hunk 1]
```diff
@@ -23,7 +23,8 @@ public struct ProblematicEventInfo {
                 startTimestamp: Date,
                 endTimestamp: Date,
                 encryptedAssociatedData: Bytes,
-                cipherTextNonce: Bytes) {
+                cipherTextNonce: Bytes)
+    {
```
Comment (ubfelix) [inline Sources/CrowdNotifierBaseSDK/Models/ProblematicEventInfo.swift]: Was ist das für eine Regel? Die Klammer unten verwirrt mich etwas..

File: Sources/CrowdNotifierBaseSDK/Models/VenueInfo.swift [hunk 2]
```diff
@@ -34,7 +34,8 @@ public struct VenueInfo: Codable {
                 validFrom: Int,
                 validTo: Int,
                 qrCodePayload: Data?,
-                countryData: Data) {
+                countryData: Data)
+    {
```
Comment (ubfelix) [inline Sources/CrowdNotifierBaseSDK/Models/VenueInfo.swift]: Same here

File: Sources/CrowdNotifierBaseSDK/Models/ProblematicEventInfo.swift (hunk 1 above)
Comment (UBaggeler) [reply Sources/CrowdNotifierBaseSDK/Models/ProblematicEventInfo.swift]: >`wrapMultilineStatementBraces`
> Wrap the opening brace of multiline statements.

https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapmultilinestatementbraces

File: Sources/CrowdNotifierBaseSDK/Models/VenueInfo.swift (hunk 2 above)
Comment (UBaggeler) [reply Sources/CrowdNotifierBaseSDK/Models/VenueInfo.swift]: Same here

>`wrapMultilineStatementBraces`
> Wrap the opening brace of multiline statements.

https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapmultilinestatementbraces

File: Sources/CrowdNotifierSDK/Crypto/CryptoUtils.swift [hunk 3]
```diff
@@ -8,16 +8,15 @@
...
+import Foundation
 import HKDF
+import libmcl
 
 final class CryptoUtils {
-
     private static let NONCE_LENGTH: Int = 32
 
-    private struct DomainKeys {
+    private enum DomainKeys {
```
Comment (UBaggeler) [inline Sources/CrowdNotifierSDK/Crypto/CryptoUtils.swift:19]: > enumNamespaces
> Converts types used for hosting only static members into enums.

https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#enumnamespaces

**Review by ubfelix — APPROVED** [review]

**Review by zimmermannubique — APPROVED** [review]

**Review by ubfelix — APPROVED** [review]

**Pull request merged**